eslint.config.js

```diff
@@ -37,6 +41,52 @@ export default [
           varsIgnorePattern: '^_',
           caughtErrorsIgnorePattern: '^_'
         }
+      ],
+      'import/order': [
+        'error',
+        {
+          'groups': [
+            'builtin', // node built-in modules
+            'external', // npm packages
+            'internal', // @/ and ~/ aliases
+            ['parent', 'sibling', 'index'] // relative imports
+          ],
+          'pathGroups': [
+            {
+              pattern: 'react',
+              group: 'external',
+              position: 'before'
+            },
+            {
+              pattern: 'react-router',
+              group: 'external',
+              position: 'before'
+            },
+            {
+              pattern: '@interledger/**',
+              group: 'external',
+              position: 'after'
+            },
+            {
+              pattern: '@/components{,/**}',
+              group: 'internal',
+              position: 'after'
+            },
+            {
+              pattern: '@/**',
+              group: 'internal'
+            },
+            {
+              pattern: '~/**',
+              group: 'internal'
+            }
+          ],
```

Comment:
```suggestion
          ],
          'pathGroupsExcludedImportTypes': ['builtin'],
```

Feel free to test, but found this add would make it to where React rules work and prioritize to top